pytest_tests/steps/s3_gate_base.py

```diff
@@ -65,11 +66,20 @@ def s3_client(self, prepare_wallet_and_deposit, client_shell: Shell, request):
         TestS3GateBase.wallet = wallet
 
 
+def get_wallet_pass(hosting: Hosting, node_id=1):
+    service_configs = hosting.find_service_configs(S3_GATE_SERVICE_NAME_REGEX)
+    wallet_password = service_configs[node_id - 1].attributes.get("wallet_password")
+    return wallet_password
+
+
 @allure.step("Init S3 Credentials")
-def init_s3_credentials(wallet_path: str, s3_bearer_rules_file: Optional[str] = None):
+def init_s3_credentials(
+    wallet_path: str, hosting: Hosting, s3_bearer_rules_file: Optional[str] = None
+):
     bucket = str(uuid.uuid4())
     s3_bearer_rules = s3_bearer_rules_file or "robot/resources/files/s3_bearer_rules.json"
-    gate_public_key = get_wallet_public_key(S3_GATE_WALLET_PATH, S3_GATE_WALLET_PASS)
+    s3_password = get_wallet_pass(hosting)
```

Comment:
Currently there is no control which node developer wants to get pass from.
So if someday passwords will be different on different nodes and we want to do requests to different node this code should be modified again